doc/client_lua_api.md

```diff
@@ -774,6 +774,37 @@ An interface to manipulate minimap on client UI
 * `get_mode()`: returns the current minimap mode
 * `toggle_shape()`: toggles minimap shape to round or square.
 
+### LocalPlayer
```

Comment:
you should document how to get it here as well. Don't necessary take examples from the rest of the docs